application/src/main/resources/ecc.yml

```diff
@@ -27,16 +27,23 @@ connection:
     host: localhost
     port: 9042
     ##
-    ## Connection Timeout for CQL.
+    ## Connection Timeout for an CQL attempt.
     ## Specify a time to wait for cassandra to come up.
-    ## Connection is tried every five second until either the timeout time passes or the connection is successful.
+    ## Connection is tried based on retry policy delay calculations. Each connection attempt will use timeout to calculate CQL Connection proccess delay.
     ##
     timeout:
       time: 60
       unit: seconds
     retryPolicy:
+      ## Max number of attempts that ecChronos will try to connect with Cassandra.
       maxAttempts: 5
+      unit: seconds
+      ## Delay use to wait between an attempt and another, this value will be multiplied by the current attempt count.
+      ## If the calculated delay is greater than maxDelay, maxDelay will be used instead of the calculated delay
```

Comment:
This is not correctly described, right? You do a power of 2 to the attempt, and then multiply the delay value to it. As in:
myDelay * Math.pow(2, attempt)

Missing a period at the end.